docs/src/main/sphinx/static/trino.css

```diff
@@ -81,15 +74,6 @@ pre.literal-block {
     font-size: unset;
 }
 
-@media only screen and (min-width: 88.25em) {
```

Comment:
This might be the issue that hides the right sidebar fyi @wendigo @kumiDa  .. but thats just a guess..

Reply:
I too had a hunch that this could be the issue, but verified that the sidebar margin was not causing the issue.
So, this possibility of cause can be struck out.